sdk/hash/src/lib.rs

```diff
@@ -1,4 +1,5 @@
 #![no_std]
+#![cfg_attr(docsrs, feature(doc_auto_cfg))]
```

Comment:
Can you also add the change to `sdk/hash/Cargo.toml` for `all-features` and `rustdoc-args`?